ocs_ci/utility/aws.py

```diff
@@ -1788,17 +1788,17 @@ def delete_cluster_buckets(cluster_name):
         r = re.compile(pattern)
         filtered_buckets = list(filter(r.search, bucket_names))
         logger.info(f"Found buckets: {filtered_buckets}")
-        if len(filtered_buckets) == 1:
+        if len(filtered_buckets):
```

Comment:
Since we are iterating over `filtered_buckets` I don't think we need to check the length anymore. The only thing this really saves us from doing is creating an s3_resource object when no buckets exist, but that shouldn't be a big deal.

Reply:
Addressed by 5ff666b.